build.sh

```diff
@@ -33,12 +33,19 @@ elif [ ! -z "$DOCKERCFG" ]; then
 	print_msg "   Detected configuration in \$DOCKERCFG"
 	echo "$DOCKERCFG" > /root/.dockercfg
 	unset DOCKERCFG
+elif [ ! -z "$DOCKER_CONFIG" ]; then
+	print_msg "   Detected configuration in \$DOCKER_CONFIG"
+	if [ ! -d /root/.docker ]; then
+		mkdir /root/.docker
+	fi
+	echo "$DOCKER_CONFIG" > /root/.docker/config.json
+	unset DOCKER_CONFIG
 elif [ ! -z "$USERNAME" ] && [ ! -z "$PASSWORD" ]; then
 	REGISTRY=$(echo $IMAGE_NAME | tr "/" "\n" | head -n1 | grep "\." || true)
 	print_msg "   Logging into registry using $USERNAME"
 	docker login -u $USERNAME -p $PASSWORD -e ${EMAIL-no-email@test.com} $REGISTRY
 else
-	print_msg "   WARNING: no \$USERNAME/\$PASSWORD or \$DOCKERCFG found - unable to load any credentials for pushing/pulling"
+	print_msg "   WARNING: no \$USERNAME/\$PASSWORD or \$DOCKERCFG or \$DOCKER_CONFIG found - unable to load any credentials for pushing/pulling"
```

Comment:
I'd change it for `WARNING: no \$USERNAME/\$PASSWORD or \$DOCKER_CONFIG found - unable to load any credentials for pushing/pulling`
No need to display the backwards compatibility one